src/Components/Dialogs/GetItems/getItem.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import style from './getItem.module.css';
+
+const getItem = (props) => {
+  const numbers = props.nambers;
```

Comment:
There's a typo in "props.nambers" - it should be "props.numbers"

Reply:
Yes,I pressed the wrong button